### PR TITLE
Add color

### PR DESCRIPTION
**Pull request by usergenic**
Now there's color.

### CODE REVIEW

**Comment by usergenic** [comment]
Please note this includes #45 because i branched from it.

File: package.json [hunk 1]
```diff
@@ -28,8 +28,10 @@
     "url": "https://github.com/PolymerLabs/tattoo/issues"
   },
   "dependencies": {
+    "@types/colors": "^0.6.33",
     "bottleneck": "^1.10.2",
     "bower": "^1.7.7",
+    "colors": "^1.1.2",
```
Comment (FredKSchott) [inline package.json]: we've been using chalk everywhere else in our ecosystem, it would be good to standardize on that

File: src/bin/tattoo.ts [hunk 2]
```diff
@@ -24,18 +24,24 @@ async function main() {
     mergeConfigFileOptions(cliOptions, loadConfigFileOptions(cliOptions));
     ensureGitHubToken(cliOptions);
     const runnerOptions: RunnerOptions = {
+      color: cliOptions['color'],
```
Comment (FredKSchott) [inline src/bin/tattoo.ts]: once we move into the programatic runner interface, lets drop the 'on'/'off' in favor of true/false for the `color` option. Then you don't need to translate to a boolean inside the runner.

File: src/runner.ts [hunk 3]
```diff
@@ -99,6 +105,7 @@ export class Runner {
   // TODO(usergenic): This constructor is getting long.  Break up some of
   // these stanzas into supporting methods.
   constructor(options: RunnerOptions) {
+    this._color = options.color === 'on';
```
Comment (rictic) [inline src/runner.ts]: Super minor: the description of the argument suggests that the color is on unless the option is given as `'off'`. Consider:

```typescript
this._color = options.color !== 'off';
```

Comment (usergenic) [inline src/runner.ts]: Fair enough.  Inverted as suggested.

**Pull request merged**